### ISSUE TITLE
tabs: extra bottom border caused by backticks

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#1260]
v2.14.1 (I'm using the latest master though)

Note the extra bottom border in the second version:

![image](https://user-images.githubusercontent.com/1673303/85555678-ba779d80-b658-11ea-8dba-66d20d62f270.png)

![image](https://user-images.githubusercontent.com/1673303/85555780-d5e2a880-b658-11ea-89e4-eda321864907.png)

At least in this case it seems to be caused by backticks around `.py` (in the second tab heading) based on my trial and error experiments.

It's just a cosmetic issue but it does affect the visual meaning of the component as it makes harder to spot which tab is being displayed.

PS: I haven't been able to isolate this problem into a single reproducible cause. To observe the problem:
1. run the MarkBind site https://github.com/nus-cs2103-AY1920S2/website
2. See http://127.0.0.1:8080/website/admin/tp-deliverables.html#pe-d-during-the-session
    ![image](https://user-images.githubusercontent.com/1673303/85923770-6b33a600-b8c0-11ea-9f38-f7c2e3b9ce55.png)
3. See http://127.0.0.1:8080/website/admin/tp-deliverables.html#pe-phase-1-part-i-product-testing-60-minutes-2 (same code included in another place, and the problem doesn't appear there!)
   ![image](https://user-images.githubusercontent.com/1673303/85923779-91594600-b8c0-11ea-9651-7463edbc3ee1.png)

Based on a git bisect, this started happening in 5db6351df9898ac2cfc958181ce694abe756a2d3

**Comment by damithc** [comment]
Looks like it's not just the backticks. In one case, I was able to get rid of the extra border by adding a blank line below a `<tab>` tag.
In another case, this absolutely simple code causes the problem.
```html

<tabs>
<tab header="Java">

x

</tab>
<tab header="Python">

y

</tab>
</tabs>
```

![image](https://user-images.githubusercontent.com/1673303/85922594-3707b780-b8b7-11ea-829a-a56e2f956f86.png)

Funny thing is, the same code is reused in another place using `<include>` and it works fine.
![image](https://user-images.githubusercontent.com/1673303/85922617-64ecfc00-b8b7-11ea-8514-d2acb02787d4.png)

It looks like the cause is the cumulative effect of surrounding code rather than the code inside the tabs component alone.

Also note that the problem doesn't exist at v2.14.1 released version. It's something introduced since then.

**Comment by ang-zeyu** [comment]
it works fine for me, even with the inline code in the header, could you post the full markup?

**Comment by damithc** [comment]
Based on a git bisect, this started happening in 5db6351df9898ac2cfc958181ce694abe756a2d3

@ang-zeyu I'm unable to extract out a code fragment to reproduce the problem. Need to follow the steps I added to the description to observe it.

**Comment by damithc** [comment]
I can't reproduce it anymore. May be got fixed by a recent commit. Will reopen if it shows up again.

**Comment by damithc** [comment]
I have a live example [here](https://nus-cs2103-ay2021s1.github.io/website/schedule/week2/topics.html#project-management-revision-control-saving-history)

![image](https://user-images.githubusercontent.com/1673303/88281481-53faa380-cd1a-11ea-9c8e-bcc8a24bbfba.png)

Doesn't seem to happen often though.

**Comment by damithc** [comment]
Would be good to fix before the new semester though, as it affects the textbook.

**Comment by ang-zeyu** [comment]
![Untitled](https://user-images.githubusercontent.com/3306138/88283830-cc636380-cd1e-11ea-865f-afb0b4ac5955.png)

👀 

displays like such on my computer. also tested on firefox and chrome mobile.

It's likely due to the way browsers render things across systems. We can leave this open to keep track until we migrate to bootstrap vue for tabs which hopefully fixes it

**Comment by damithc** [comment]
Yes, looks like browser dependent. I'm using Chrome desktop. Even within the same page, it happens to some tabs but not others.

**Comment by tlylt** [comment]
It seems like an element sizing/margin/css issue.

> I have a live example [here](https://nus-cs2103-ay2021s1.github.io/website/schedule/week2/topics.html#project-management-revision-control-saving-history)

I am able to reproduce it (aka making the line appear) by changing the zoom level of the browser to 90%, and it disappears at other zoom levels. I am using Google Chrome. This reproduction is not very consistent, so revisiting the page again and I found that I needed to change the zoom to 50% for it to appear. However, adjusting and readjusting the zoom level seems always to be able to trigger it.

(Appear)
![image](https://user-images.githubusercontent.com/41845017/204695771-cc43a7d9-30f7-4bce-9f19-e8fa656e37c3.png)

(Diappear)
![image](https://user-images.githubusercontent.com/41845017/204695952-eec39221-74a2-473f-aa46-33ec4e329e53.png)

**Comment by damithc** [comment]
Thanks for the investigation @tlylt 
Yes, it appears intermittently.
